static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -109,6 +109,10 @@
     .adverts__row > * + *::before {
         content: none;
     }
+
+    .adverts__badge__help {
```

Comment:
Yes we can. I thought about it but I've chicken out because it could be useful for something else. But yeah, lets go for it.